apps/e2e-tests/tests/example.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, test } from '@playwright/test';
+import { appName, urls } from '../configuration';
+import { checkIfAppIsAvailable } from '../utils/check-if-app-is-available';
+import { installTheApp } from '../utils/install-the-app';
+import { logInIntoDashboard } from '../utils/log-in-into-dashboard';
+import { randomString } from '../utils/random-string';
+
+test("The app can be installed", async ({page}) => {
+  // TODO: add function to completely remove all the installed apps in the test env
```

Comment:
we need to have our own controlled env, we can prepare it before tests and prune all apps